modules/qualitycontrol/SendVideoController.js

```diff
@@ -80,23 +107,24 @@ export class SendVideoController {
      * Selects the lowest common value for the local video send constraint by looking at local user's preference and
      * the active media session's receive preference set by the remote party.
      *
+     * @param {string} sourceName - The source for which sender constraints have changed.
      * @returns {number|undefined}
      */
-    selectSendMaxFrameHeight() {
-        const activeMediaSession = this.conference._getActiveMediaSession();
+    selectSendMaxFrameHeight(sourceName = null) {
```

Comment:
again, as in the other comment I think it may be better to have a duplicate version that works with source names